src/main/asciidoc/appendix/settings.adoc

```diff
@@ -170,7 +170,7 @@ The following plugins are available for ArcadeDB Server:
 |`asyncOperationsQueueImpl`|Queue implementation to use between 'standard' and 'fast'. 'standard' consumes less CPU than the 'fast' implementation, but it could be slower with high loads|String|standard
 |`asyncOperationsQueueSize`|Size of the total asynchronous operation queues (it is divided by the number of parallel threads in the pool)|Integer|1024
 |`asyncTxBatchSize`|Maximum number of operations to commit in batch by async thread|Integer|10240
-|`asyncWorkerThreads`|Number of asynchronous worker threads. 0 (default) = available cores minus 1|Integer|0
+|`asyncWorkerThreads`|Number of asynchronous worker threads. 0 (default) = available cores minus 1, but at least 1|Integer|0
```

Comment:
The last column is still to 0

Reply:
I understood that `0` is the default value and a special value meaning max(1, num_cores - 1), isn't it?

Reply:
This is how it's defined:

```java
  ASYNC_WORKER_THREADS("arcadedb.asyncWorkerThreads", SCOPE.DATABASE,
      "Number of asynchronous worker threads. 0 (default) = available cores minus 1", Integer.class,
      Runtime.getRuntime().availableProcessors() > 1 ? Runtime.getRuntime().availableProcessors() - 1 : 1),
```

The last parameter `Runtime.getRuntime().availableProcessors() > 1 ? Runtime.getRuntime().availableProcessors() - 1 : 1` set the available cores - 1 or just 1. This algo is the default.

Reply:
Sorry, I am still unsure to what I should change the last column to.